experimental/packages/opentelemetry-exporter-metrics-otlp-grpc/test/OTLPMetricExporter.test.ts

```diff
@@ -182,15 +182,15 @@ const testOTLPMetricExporter = (params: TestParams) =>
           headers: {
             foo: 'bar',
           },
-          aggregationTemporality: AggregationTemporality.CUMULATIVE
+          preferredAggregationTemporality: AggregationTemporality.CUMULATIVE
```

Comment:
If the preferred aggregation temporality is removed what is this now doing?

Reply:
As OTLP exporters should be configurable with either temporalities and the configuration env var for it will be [OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md#:~:text=kind%20using%20the-,OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE,-variable%20as%20described), the idea was to bring some similarity into the naming. Perhaps `temporalityPreference` would suit better?

Reply:
+1 for `temporalityPreference` as it aligns the naming.

Reply:
Renamed it to `temporalityPreference`